src/main/kotlin/study/racing/domain/Distance.kt

```diff
@@ -0,0 +1,14 @@
+package study.racing.domain
+
+class Distance {
+
+    private var moveDistance: Int = 0
```

Comment:
```suggestion
    var moveDistance: Int = 0
    private set
```

private set을 사용해주면 어떠한 장점이 있을까요?

Reply:
외부에서 moveDistance 의 값을 수정할 수 가 있는 상태였네요.
이걸 방지하기 위해 private set을 선언해줘야 하네요.
감사합니다.

Reply:
객체 내부의 변수를 외부에서 수정하지 못하게 하기 위해서 private로 접근을 제어 했었습니다.
이렇게 하면 단점으로 Distance().moveDistance 같이 해당 필드를 접근할 수가 없게 되므로 불필요한 getter를 생성해야 했습니다.

알려주신 방식을 적용하면 setter를 private 해줌과 동시에 객체의 변수를 접근할 수 있어서 불필요한 getter를 제거할 수 있을 것 같아요.